packages/tools/bcm2835-bootloader/package.mk

```diff
@@ -22,7 +22,17 @@ makeinstall_target() {
 
     find_file_path config/dt-blob.bin && cp -PRv $FOUND_PATH $INSTALL/usr/share/bootloader
 
-    cp -PRv $PKG_DIR/scripts/update.sh $INSTALL/usr/share/bootloader
+    if [ -f $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader/update.sh ]; then
+      cp -av $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader/update.sh $INSTALL/usr/share/bootloader/
+    elif [ -f $PROJECT_DIR/$PROJECT/bootloader/update.sh ]; then
+      cp -av $PROJECT_DIR/$PROJECT/bootloader/update.sh $INSTALL/usr/share/bootloader/
+    fi
+
+    if [ -f $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader/canupdate.sh ]; then
+      cp -av $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader/canupdate.sh $INSTALL/usr/share/bootloader/
+    elif [ -f $PROJECT_DIR/$PROJECT/bootloader/canupdate.sh ]; then
+      cp -av $PROJECT_DIR/$PROJECT/bootloader/canupdate.sh $INSTALL/usr/share/bootloader/
+    fi
```

Comment:
Shouldn't we use `find_file_path bootloader/update.sh` and `find_file_path bootloader/canupdate.sh` (same as lines above & below)?

Reply:
Now using find_file_path. Cheers